src/aura/BDI_ManageAdvancedMapping/BDI_ManageAdvancedMappingHelper.js

```diff
@@ -0,0 +1,46 @@
+({
+    isSysAdmin : function(component) {
+        let userId = 'asdfsdf';
+
+        let action = component.get('c.isAdminUser');
+        action.setParams({
+            'userId' : userId
+        });
+
+        action.setCallback(this, function(response) {
+            let state = response.getState();
+            const SUCCESS = 'SUCCESS';
+            const ERROR = 'ERROR';
+
+            if(state === SUCCESS) {
```

Comment:
```suggestion
            if (state === SUCCESS) {
```

Could we add a single space between the if and parenthesis? There are a few of these.

Reply:
@psadie-sfdo Will do.

Reply:
fixed!